src/mlpack/methods/naive_bayes/naive_bayes_classifier_impl.hpp

```diff
@@ -162,8 +162,7 @@ void NaiveBayesClassifier<MatType>::Train(const VecType& point,
 }
 
 template<typename MatType>
-void NaiveBayesClassifier<MatType>::Classify(const MatType& data,
-                                             arma::Row<size_t>& results)
```

Comment:
The user may not always want the probabilities, so we should leave the other overload in place and add a new method, instead of simply changing the signature of the existing method (which also breaks reverse compatibility).